app/views/users/profile/show.html.erb

```diff
@@ -45,3 +45,13 @@
 <p><%= link_to "Edit", "/profile/edit" %> | <%= link_to "Delete", destroy_user_path(@user), method: :delete, data: { confirm: "You sure?" } %></p>
 
 <p><%= link_to "Change your password", edit_user_registration_path %></p>
+
+<h4>Your Recent Activity</h4>
```

Comment:
Cool. We'll tackle later/at another time exactly how to show this information to each user.